src/pixbuf_animation_iter.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013-2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use super::Pixbuf;
+use gdk_pixbuf_sys;
+use glib::translate::*;
+use glib::TimeVal;
+
+glib_wrapper! {
+    pub struct PixbufAnimationIter(Object<gdk_pixbuf_sys::GdkPixbufAnimationIter, PixbufAnimationIterClass>);
+
+    match fn {
+        get_type => || gdk_pixbuf_sys::gdk_pixbuf_animation_iter_get_type(),
+    }
+}
+
+impl PixbufAnimationIter {
+    pub fn advance(&self, start_time: TimeVal) -> bool {
```

Comment:
```suggestion
    pub fn advance(&self, start_time: &TimeVal) -> bool {
```

Reply:
`TimeVal` is `Copy`, no?

Reply:
Yes

Reply:
Do you still suggest we should take a reference then? XD

Reply:
No :)